cocos/base/CCDirector.cpp

```diff
@@ -1076,7 +1076,7 @@ void Director::showStats()
     static unsigned long prevCalls = 0;
     static unsigned long prevVerts = 0;
     static float prevDeltaTime  = 0.016; // 60FPS
-    static const float FPS_FILTER = 0.05;
+    static const float FPS_FILTER = 0.10;
```

Comment:
@ricardoquesada, do you know why you added `FPS_FILTER` and what does it used for?